src/screens/FeedScreen/ContentPoll.js

```diff
@@ -243,14 +244,14 @@ const stylesComponent = (pollLength) =>
       marginBottom: 0
     },
     pollOptionItemContainer: {
-      backgroundColor: COLORS.lightgrey,
+      backgroundColor: colors.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.lightgrey` to `colors.lightgrey`. Make sure that `colors.lightgrey` is defined and is the intended color.
```diff
- backgroundColor: COLORS.lightgrey
+ backgroundColor: colors.lightgrey
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->